Russian localization updates

Kostenkov-2021 opened the pull request:
Russian nvda.po file has been updated due to the recent changes.

Kostenkov-2021 committed:
Russian localization and documentation

Russian nvda.po and readme.md files has been added

Kostenkov-2021 committed:
Merge branch 'Nael-Sayegh:main' into main

Kostenkov-2021 committed:
Addition of Russian changelog

Russian changelog.md has been added.

Kostenkov-2021 committed:
Updates in changelog_ru.md

The location of changelog_ru.md file has been changed

Kostenkov-2021 committed:
Changelog updates

changelog.md file has been updated.

Kostenkov-2021 committed:
Localization fixes

Russian nvda.po file has been fixed.

Kostenkov-2021 committed:
Merge branch 'Nael-Sayegh:main' into main

Kostenkov-2021 committed:
Localization and changelog updates

Russian, Czech and changelog has been fixed.

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Localization updates

Czech, French and Portuguese nvda.po files has been updated according to last changes.

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Localization and documentation changes

* Czech nvda.po has been fixed to avoid poedit errors.
* Russian readme.md has been updated according to the last version of original readme.

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Adding add-on description

Add-on description has been added in Russian and Portuguese nvda.po files.

Kostenkov-2021 committed:
Fixes in French translation

The error with space in French nvda.po file has been fixed.

Nael-Sayegh committed:
Restore French po file

Nael-Sayegh committed:
Update action for your PR

Nael-Sayegh committed:
Modify action too Accept your PR

Nael-Sayegh committed:
update action

Kostenkov-2021 committed:
Restoring French localization

French nvda.po has been restored.

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Merge branch 'Nael-Sayegh:main' into main

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Russian documentation updates

Russian readme.md file has been updated.

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Merge branch 'Nael-Sayegh:main' into main

Kostenkov-2021 committed:
Merge branch 'Nael-Sayegh:main' into main

Kostenkov-2021 committed:
Russian localization updates

Russian nvda.po file has been updated.

Nael-Sayegh commented:
@Kostenkov-2021 Can you also update the readme.md file?

Kostenkov-2021 commented:
Hello. Yes, I can update the documentation.

Kostenkov-2021 committed:
Russian documentation updates

Russian readme.md file has been updated.

Nael-Sayegh commented:
@Kostenkov-2021 Thank you for the update. The sentence indicating that Scanvox is located in the tools menu also needs to be changed to say that it is in the main menu.

Nael-Sayegh commented:
@Kostenkov-2021 The translation .po file is no longer present, there is only the documentation file with the second modification pending.

Nael-Sayegh committed:
Version 2024.07.02 (#50)

* ci(deps): pre-commit autoupdate (#46)

* ci(deps): pre-commit autoupdate (#38)

updates:
- [github.com/astral-sh/ruff-pre-commit: v0.4.3 → v0.4.4](https://github.com/astral-sh/ruff-pre-commit/compare/v0.4.3...v0.4.4)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

* ci(deps): pre-commit autoupdate (#41)

updates:
- [github.com/astral-sh/ruff-pre-commit: v0.4.4 → v0.4.5](https://github.com/astral-sh/ruff-pre-commit/compare/v0.4.4...v0.4.5)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

* Version 2024.06.01 (#36)

* Updating readme.md

* Updating workflow

* Updating workflow

* Updating Turkish translation by umut korkmaz (#34)

* Updating documentation

* Updating French translation (#35)

* Updating add-on description and updating French translation

* Updating French translation

* Updating Turkish po file

* Updating all readme.md (#37)

* updating Czech translation (#39)

* Updating Czech translation

* Updating readme.md

* Adding the German translation (#40)

* Updating all readme.md files

* Updating buildvars.py

* Updating link in readme files

* ci(deps): pre-commit autoupdate (#42)

updates:
- [github.com/astral-sh/ruff-pre-commit: v0.4.5 → v0.4.7](https://github.com/astral-sh/ruff-pre-commit/compare/v0.4.5...v0.4.7)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

* ci(deps): pre-commit autoupdate

updates:
- [github.com/astral-sh/ruff-pre-commit: v0.4.7 → v0.4.8](https://github.com/astral-sh/ruff-pre-commit/compare/v0.4.7...v0.4.8)

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
Co-authored-by: Nael Sayegh <61361779+Nael-Sayegh@users.noreply.github.com>

* Fixed an issue where page numbers continued to increase after clicking delete (#49)

* Correction of pagination of pages after clicking the delete button

* Correcting the name of calling class

* Adding textmanage in class parameter for delete

* Updating changelog

* Updating all translations (#51)

* Updating all translations

* Updating readme file

* ci(deps): pre-commit autoupdate (#48)

updates:
- [github.com/astral-sh/ruff-pre-commit: v0.4.7 → v0.5.0](https://github.com/astral-sh/ruff-pre-commit/compare/v0.4.7...v0.5.0)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
Co-authored-by: Nael-Sayegh <naelousay@gmail.com>

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

Nael-Sayegh committed:
Change base branch for pre-commit-ci

Kostenkov-2021 committed:
Merge branch 'main' of https://github.com/Kostenkov-2021/scanvox-for-nvda

Kostenkov-2021 committed:
Localization and documentation fixes.
